appveyor.yml

```
@@ -54,6 +54,14 @@ for:
     - 7z a "bin\dhall-text-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-text.exe"
     - 7z a "bin\dhall-bash-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-bash.exe"
     - 7z a "bin\dhall-lsp-server-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall-lsp-server.exe"
+
+  # stack is not able to build test dependencies with lts-6
+  test_script:
+    - stack test dhall:tasty
+    - stack test dhall-json
+    - stack test dhall-text
+    - stack test dhall-bash
+    - stack test dhall-lsp-server
```

If @Gabriel439 agrees:

```suggestion
    # - stack test dhall-lsp-server # Disabled while the tests are broken.
```

---

Yeah, I think it's fine to disable these tests for now